bigquery/README.md

```diff
@@ -1,19 +1,16 @@
 ## BigQuery indexer
 
-Follow the instructions below to create an Elasticsearch index in an
-Elasticsearch docker container. Note that we run indexer.py directly (not using
-docker) because it's trickier to authenticate to Google Cloud Platform from
-within docker.
-
 ### Quickstart
 
+Index a BigQuery table into an Elasticsearch container on your local machine.
+
 * If you want to use a [sample public dataset](https://bigquery.cloud.google.com/table/google.com:biggene:platinum_genomes.sample_info):
   * [Install bq](https://cloud.google.com/bigquery/docs/bq-command-line-tool#installation)
```

Comment:
Discussed offline, I'm starting to leave out these install steps because it's easy to google "install gcloud", in the interest of decluttering the README.